cartridge/issues.lua

```diff
@@ -163,17 +163,18 @@ local function list_on_instance(opts)
                 )
             }
             table.insert(ret, issue)
-        elseif upstream.idle > box.cfg.replication_timeout then
-            -- A replica sends heartbeat messages to the master
-            -- every second, and the master is programmed to
-            -- reconnect automatically if it does not see heartbeat
-            -- messages within replication_timeout seconds.
-            --
-            -- Therefore, in a healthy replication setup, idle
-            -- should never exceed replication_timeout: if it does,
-            -- either the replication is lagging seriously behind,
-            -- because the master is running ahead of the replica,
-            -- or the network link between the instances is down.
+        elseif upstream.idle > 2 * box.cfg.replication_timeout then
+            -- https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-replication-replication-timeout
+
+            -- Since version 1.8.2. If the master has no updates to send to the
```

Comment:
Where did you find Tarantool 1.8?

Reply:
I only cite the doc (in case the doc link rots).

Reply:
Anyway, I guess it's not that important

Reply:
Nevermind, I've read it again. I didn't get it right. Fixed